cypress/e2e/requests/edit-request.cy.js

```diff
@@ -45,7 +45,6 @@ describe('ui-requests: Request: Edit requests. Make sure that edits are being sa
       cancelledDate: new Date().toISOString(),
```

Comment:
'oldRulesText' is assigned a value but never used. Allowed unused vars must match /React/u

Reply:
Deleted